feature/FI-1610-options-display

AlyssaWang opened the pull request:
Adds selected options to the header.

<img width="424" alt="Screen Shot 2022-07-13 at 12 48 06 PM" src="https://user-images.githubusercontent.com/11209247/178787815-f5e3d908-8ac5-4ce4-8cb2-0856855fb5c8.png">

***Note:*** Full option choice data isn't attached to the `testSession.suite_options` object so there's a bit of parsing to pull that info (specifically labels) directly from the `testSuite.suite_options` object. We might want to look into making that data more widely accessible.

chris-at-work reviewed (approved):
LGTM

TIL those `warning` and `error` palette colors are available, maybe I should have used those for some icons that I changed.

AlyssaWang commented:
> TIL those `warning` and `error` palette colors are available, maybe I should have used those for some icons that I changed.

@chris-at-work I think I updated those values in the accessibility PRs :)
